google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -118,6 +118,35 @@ def table_ref
           table_ref
         end
 
+        ###
+        # Is the table partitioned?
+        #
+        # @!group Attributes
+        #
+        def time_partitioning?
+          !@gapi.time_partitioning.nil?
+        end
+
+        ###
+        # The period for which the table is partitioned, if any.
+        #
+        # @!group Attributes
+        #
+        def time_partitioning_type
+          ensure_full_data!
+          time_partitioning.type if time_partitioning?
+        end
+
+        ###
+        # The expiration for the table partitions, if any,  in milliseconds.
+        #
+        # @!group Attributes
+        #
+        def time_partitioning_expiration
+          ensure_full_data!
+          time_partitioning.expiration_ms if time_partitioning?
```

Comment:
`time_partitioning. expiration_ms ` needs to be changed to `@gapi.time_partitioning. expiration_ms `.